src/health_misinfo_shared/fine_tuning.py

```diff
@@ -128,6 +181,30 @@ def make_training_set_simple() -> pd.DataFrame:
     return training_data_final_df
 
 
+def calculate_claim_summary_score(labels: Dict[str, str]) -> str:
+    total_score = 0
+    for category in [
+        "understandability",
```

Comment:
Are these just the keys from the `LABEL_SCORES` dict? If so, would be simpler to read from that rather than duplicate the list here.